Makefile

```diff
@@ -4,18 +4,37 @@ help:
 	@echo ""
 	@echo "  Targets"
 	@echo ""
+	@echo "    models       Download and extract models to $(PWD)/models_eynollah"
```

Comment:
Nice to have this covered in the Makefile so that you can `make models`!

(note: we should probably harmonize this to `make models` (plural) for [sbb_binarization](https://github.com/qurator-spk/sbb_binarization/blob/ea69494d2aa66255491e7f2facb2f9b50c7b342e/Makefile#L11) as well.